packages/components/src/components/DiagramSequence.vue

```diff
@@ -35,6 +35,7 @@
               :actionSpec="action"
               :collapsed-actions="collapsedActions"
               :key="actionKey(action)"
+              :appMap="appMap"
```

Comment:
At this top level, passing an AppMap seems fine.

Reply:
Moved everything to parent.